Data Structures/Linked-Lists/list-implementation.cpp

```diff
@@ -303,8 +303,31 @@ struct node *splice(struct node *head,struct node **end, int start, int endd)
     
 }
 
-//the main function to run the code
+//function to check palindrome, first recursion till last element
+//then start comparing with element at start pointer
+int pa_check(node *tem,int con,node *&ch){
+  if(tem==NULL)
```

Comment:
@abhishek0220 Can you please add proper spaces between `=` assignment statement as mentioned in the [Coding standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards) ?